docs/rfcs/0001-rfc-process.md

```diff
@@ -0,0 +1,63 @@
+- Status: Proposed
+- Date: 2023-05-23
+- RFC PR: TODO
+
```

Comment:
Even if this is private, I wonder if we want to add a link to Jira/Github too

Reply:
Good idea, I think a link to an issue would be nice to track the implementation of the RFC, if applicable.